lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback_file.inc

```diff
@@ -169,6 +169,26 @@ class ReportbackFileEntityController extends EntityAPIController {
   }
 
   /**
+   * Overrides save() method.
+   *
+   * Updates count variables for corresponding nodes and taxonomy terms.
+   */
+  public function save($entity, DatabaseTransaction $transaction = NULL) {
+    if (DOSOMETHING_REPORTBACK_LOG) {
+      watchdog('dosomething_reportback_file', 'save:' . json_encode($entity));
+    }
+    $status = NULL;
```

Comment:
Yes, this can be a game changer, didn't know that. So this PR is good if your check passes.